state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -372,6 +372,12 @@ pub mod pallet {
 	pub type MinimumDeposit<T: Config<I>, I: 'static = ()> =
 		StorageMap<_, Twox64Concat, TargetChainAsset<T, I>, TargetChainAmount<T, I>, ValueQuery>;
 
+	/// Defines the minimum amount for a single egress i.e. *not* of a batch, but the outputs of
+	/// each individual egress within that batch.
+	#[pallet::storage]
+	pub type MinimumEgress<T: Config<I>, I: 'static = ()> =
```

Comment:
This would be set to the 600 sats for Bitcon, and 0 for the rest - just getting initial feedback first, then will do this

Reply:
We could also read these values on the FE if they're added to the ingress/egress environment RPC

Reply:
done